cli/src/semgrep/rule_match.py

```diff
@@ -209,23 +209,15 @@ def get_cli_unique_key(self) -> Tuple:
 
         Used for deduplication in the CLI before writing output.
         """
-        if self.from_transient_scan:
+        return (
+            # REVIEW: comment still relevant?
```

Comment:
Look at the blame and see what else was introduced and if it is still here. At the very least this could use some clarification.

Reply:
Still current per Vivek